pkg/minikube/tunnel/route_windows.go

```diff
@@ -47,16 +76,16 @@ func (router *osRouter) EnsureRouteIsAdded(route *Route) error {
 	klog.Infof("Adding route for CIDR %s to gateway %s", serviceCIDR, gatewayIP)
 	command := exec.Command("route", "ADD", destinationIP, "MASK", destinationMask, gatewayIP)
 	klog.Infof("About to run command: %s", command.Args)
-	stdInAndOut, err := command.CombinedOutput()
-	message := string(stdInAndOut)
-	if message != " OK!\r\n" {
-		return fmt.Errorf("error adding route: %s, %d", message, len(strings.Split(message, "\n")))
-	}
-	klog.Infof("%s", stdInAndOut)
+	stdOutAndErr, err := command.CombinedOutput()
+	message := string(stdOutAndErr)
 	if err != nil {
 		klog.Errorf("error adding Route: %s, %d", message, len(strings.Split(message, "\n")))
 		return err
 	}
+	if message != " OK!\r\n" {
+		return fmt.Errorf("error adding route: %s, %d", message, len(strings.Split(message, "\n")))
+	}
+	klog.Infof("%s", stdOutAndErr)
```

Comment:
```suggestion
	klog.Info(message)
```
While we're at it do you mind simplifying this since we already converted `message` to a string